File: app/postapi/src/main/kotlin/com/sylix/postapi/controller/RestControllerV1.java
package com.sylix.postapi.controller;

import org.springframework.web.bind.annotation.RequestMapping;

@org.springframework.web.bind.annotation.RestController
@RequestMapping("v1")
public class RestControllerV1 {

    @RequestMapping("systems/ping")
    private String ping() {
        return "hello ! World !!";
    }

}

File: app/post-api/src/main/kotlin/com/sylix/postapi/controller/RestControllerV1.kt
package com.sylix.postapi.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("v1")
class RestControllerV1 {


    @RequestMapping("systems/ping")
    fun ping(): String {
        return "hello ! World !!";
    }
}









